schemas/groups/propulsion.json

```diff
@@ -64,6 +41,12 @@
       "units": "Hz"
     },
 
+    "maxRevolutions": {
```

Comment:
It seems easier to me to write a consumer if the static design information is separated from the dynamic state information. My first thought would be to have the client pull the design data on first connection, use that to configure the display model, and then subscribe to the dynamic entries for ongoing updates. 

It would be easier and more efficient to subscribe to all dynamic values for the engine(s) (or sails or environment) at once rather than cherry-picking values out of a mix of static and dynamic ones, or subscribing to all engine data and receiving the same static data in each update.